pkg/virt-launcher/virtwrap/device/hostdevice/sriov/hostdev.go

```diff
@@ -53,7 +53,7 @@ func CreateHostDevices(vmi *v1.VirtualMachineInstance) ([]api.HostDevice, error)
 	netStatusPath := path.Join(downwardapi.MountPath, downwardapi.NetworkInfoVolumePath)
 	pciAddressPoolWithNetworkStatus, err := newPCIAddressPoolWithNetworkStatusFromFile(netStatusPath)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to create PCI address pool with network status from file: %w", err)
```

Comment:
I would wrap it in newPCIAddressPoolWithNetworkStatusFromFile close to the polling call so it's future proof

Reply:
I agree, the description of the error is known to that function and not up at this level.

Also, please add a descriptive error when we return an error due to the timeout.

Reply:
Done